src/xcode/ENA/ENA/Source/Developer Menu/Features/DMDeltaOnboardingViewController.swift

```diff
@@ -53,20 +62,38 @@ class DMDeltaOnboardingViewController: UIViewController, UITextFieldDelegate {
 		textField.translatesAutoresizingMaskIntoConstraints = false
 		textField.delegate = self
 		textField.borderStyle = .bezel
-
-		let stackView = UIStackView(arrangedSubviews: [currentVersionLabel, textField, button, resetButton])
+		
+		presentedDeltaOnboardingsLabel = UILabel(frame: .zero)
+		presentedDeltaOnboardingsLabel.translatesAutoresizingMaskIntoConstraints = false
+		presentedDeltaOnboardingsLabel.text = "Finished DeltaOnboardings"
+		presentedDeltaOnboardingsLabel.textAlignment = .center
+		
+		let stackView = UIStackView(arrangedSubviews: [currentVersionLabel, textField, button, resetButton, presentedDeltaOnboardingsLabel])
 		stackView.translatesAutoresizingMaskIntoConstraints = false
 		stackView.axis = .vertical
 		stackView.spacing = 20
-
+		
 		view.addSubview(stackView)
 		NSLayoutConstraint.activate([
 			stackView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
 			stackView.topAnchor.constraint(equalTo: view.topAnchor, constant: 100.0)
 		])
+		
+		tableView.translatesAutoresizingMaskIntoConstraints = false
+		view.addSubview(tableView)
+		tableView.translatesAutoresizingMaskIntoConstraints = false
+		tableView.topAnchor.constraint(equalTo: stackView.bottomAnchor, constant: 30.0).isActive = true
```

Comment:
Instead of writing "= true" for every constraint, its possible to put all the constraints into NSLayoutConstraint.activate([ ]). See a couple line above.